CharlieBackend.Api/Controllers/StudentGroupsController.cs

```diff
@@ -119,5 +119,21 @@ public async Task<ActionResult<StudentGroupDto>> GetStudentGroupById(long id)
 
             return foundStudentGroup.ToActionResult();
         }
+
+        /// <summary>
+        /// Gets all of the student's study group
+        /// </summary>
+        /// <response code="200">Successful return of student's study groups</response>
+        /// <response code="HTTP: 404, API: 3">Error, can not find student or student's study groups</response>
+        [SwaggerResponse(200, type: typeof(IList<StudentStudyGroupsDto>))]
+        [Authorize(Roles = "Secretary, Mentor, Admin")]
+        [HttpGet("student/{id}")]
+        public async Task<ActionResult<IList<StudentStudyGroupsDto>>> GetStudentStudyGroupsByStudentId(long id)
```

Comment:
Please move this method to students controller. In your implementation we'd have the following url:
/api/student_groups/student/{id} which is not correct. 
Let's have it here: 
/api/students/{id}/groups